ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -122,26 +110,6 @@ const MessagesHomeScreen = ({
   const { present, bottomSheet } = useMessageOpening();
 
   const isScreenReaderEnabled = useScreenReaderEnabled();
-  const isLollipopEnabled = useIOSelector(isLollipopEnabledSelector);
-  const showUnsupportedDeviceBanner =
-    isLollipopEnabled && O.isNone(publicKeyOption);
-  const unsupportedDevicesStatusComponent = showUnsupportedDeviceBanner && (
```

Comment:
In `ts/fetures/lollipop/hooks/useLollipopLoginSource.ts`, the code that handles  a missing public key tag or a missing public key is still there so is it right to removed the `unsupportedDeviceStatusComponent` here?

Reply:
I had deleted that part of the code because I knew that there are no unsupported devices, but I didn't think it was actually completely correct to have deleted that part of the code. However, I reinserted the code that shows error to the user who has an unsupported device and integrated the tests that were missing.